EpsilonDriverControls/make.sh

```diff
@@ -0,0 +1,11 @@
+if [ "$1" = "clean" ]
+then
+    (cd ../EpsilonDriverControls/ && make clean)
+    (cd Tests/ && make clean)
+    exit
+fi
+
+#!/bin/bash
```

Comment:
Shebangs need to be at the top of the file.
https://en.wikipedia.org/wiki/Shebang_(Unix)